README.md

```diff
@@ -15,122 +15,51 @@ __Currently supports Linux, macOS, and Windows.__ Please file issues if you have
 
 To use this library in your project add the following to your `Cargo.toml` file:
 
-```
+```toml
 [dependencies]
-keyring = "0.10.1"
+keyring = "0.10"
 ```
 
 This will give you access to the `keyring` crate in your code. Now you can use
 the `new` function to get an instance of the `Keyring` struct. The `new`
 function expects a `service` name and an `username` with which it accesses
 the password.
 
-You can get a password from the OS keyring with the `get_password` function.
+Passwords can be added to the keyring using the `set_password` function.  Then can then be read back using the `get_password` function, and deleted using the `delete_password` method.
```

Comment:
**suggestion:** Change `Then can then be` to `Then can be`

Reply:
ha!  good catch!  I meant to say "They can then be read back" :).  Thanks I will fix it!